PL_2_komentarze.py

```diff
@@ -0,0 +1,2722 @@
+            rezultatyklient.append((listaklientow[item], wartoscadresu11, wartoscadresu22))
+        
+        Drzewo(oknoklient, rezultatyklient, entryKlient1, entryKlient2, entryKlient3)
+        Tooltip(oknoklient, text='Potwierdź wybór klienta podwójnym kliknięciem')
+    
+        if len(rezultatyklient) == 1:
+            tupelek = rezultatyklient[0]
+            entryKlient1.delete(0, END)
+            entryKlient1.insert(END, tupelek[0])
+            entryKlient1.validate()
+            entryKlient2.delete(0, END)
+            entryKlient2.insert(END, tupelek[1])
+            entryKlient2.validate()
+            entryKlient3.delete(0, END)
+            entryKlient3.insert(END, tupelek[2])
+            entryKlient3.validate()
+            oknoklient.destroy()
+            ladujklientow()
+
+def znajdzklientaenter(event):
+    znajdzklienta()
+
+ludzie = {
+    'adarog': 0,
+    'damsre': 1,
+    "arkkla": 2,
+    "marwec": 3,
+    "micros": 4,
+    "robtom": 5,
+    "krzwen": 6,
+    "hubsta": 7,
+    "danjoz": 8,
+    "krzszr": 0,
+    "malsmu": 1,
+    "marfal": 2,
+    "robwoz": 3,
+    "agnbod": 0,
+    "jarwal": 1,
+    "tomrec": 2,
+    "sylpik": 0,
+    "wojlej": 1,
+    "arkgaj": 2,
+    "miczah": 3,
+    "piosik": 4,
+    "piomur": 0,
+    "pawsko": 1,
+    "martry": 2,
+    "joasta": 3,
+    "tomnap": 0,
+    "eugpod": 1,
+    "darken": 0,
+    "przkap": 1,
+    "tomgrz": 0,
+    "jaknec": 1,
+    "pawpud": 2
+    }
+
+
+
+def kimjestes():
+    try:
+        entryprzygotowal.current(ludzie.get(getlogin()))
+    except:
+        pass
+
+def ladujlistewezy(s):
+    if s == 'PL':
+        try:
+            wb = xlrd.open_workbook(
+                '//Srv-dokumenty/Swiadectwomat/bin/spis.xls')
+        except FileNotFoundError:
+            wb = xlrd.open_workbook('bin/spis.xls')
+            messagebox.showwarning('Nie można połączyć ze spisem węży',
+                                   'Nie udało się pobrać listy węży z folderu sieciowego. Załadowano listę z dysku lokalnego, która może zawierać nieaktualne dane.\nW razie kolejnych niepowodzeń zgłoś błąd!')
+        global ws
+        ws = wb.sheet_by_name('lista')
+        listawezy.clear()
+        for row in ws.col(0):
+            listawezy.append(row.value)
+    elif s == 'ENG':
+        try:
+            wb = xlrd.open_workbook(
+                '//Srv-dokumenty/Swiadectwomat/bin/spis_en.xls')
+        except FileNotFoundError:
+            wb = xlrd.open_workbook('bin/spis_en.xls')
+            messagebox.showwarning('Nie można połączyć ze spisem węży',
+                                   'Nie udało się pobrać listy węży z folderu sieciowego. Załadowano listę z dysku lokalnego, która może zawierać nieaktualne dane.\nW razie kolejnych niepowodzeń zgłoś błąd!')
+        ws = wb.sheet_by_name('lista')
+        listawezy.clear()
+        for row in ws.col(0):
+            listawezy.append(row.value)
+    else:
+        print('cos sie popsulo')
```

Comment:
Komunikaty błędów powinny zawierać info co jebło, a jak nie wiadomo to przynajmniej w którym momencie jebło np. `Cannot load list of hoses`.